src/policy/policy.h

```diff
@@ -27,6 +27,9 @@ static const unsigned int MAX_P2SH_SIGOPS = 15;
 static const unsigned int MAX_STANDARD_TX_SIGOPS = MAX_BLOCK_SIGOPS/5;
 /** Default for -maxmempool, maximum megabytes of mempool memory usage */
 static const unsigned int DEFAULT_MAX_MEMPOOL_SIZE = 300;
+/** Default for -blocksizeacceptlimit */
```

Comment:
It does seem odd to specify 2MB as 20.
How about making it multiples of 1MB but allowing floating point numbers so 1.5MB can be specified as 1.5?